src/windows/export.py

```diff
@@ -790,7 +790,8 @@ def titlestring(sec, fps, mess):
                 video_settings["vcodec"] = image_ext
 
         # Store updated export folder path in project file
-        get_app().updates.update_untracked(["export_path"], os.path.dirname(export_file_path))
+        settings = get_app().get_settings()
+        settings.setDefaultPath(settings.actionType.EXPORT, os.path.dirname(export_file_path))
```

Comment:
Doesn't the setDefaultPath method already handle file_paths?  i.e. by reducing them to the folder path?

Reply:
It does. I can take out the `dirname` for simplicity